spec/features/friend_invitation_spec.rb

```diff
@@ -31,10 +31,13 @@ def friend_accepts_invitation
   fill_in "Password", with: "friend_password"
   fill_in "Credit Card Number", with: "4242424242424242"
   fill_in "Security Code", with: "123"
+  select "7 - July", from: "date_month"
+  select (Time.now.year+1), from: "date_year"
   click_on "Sign Up"
 end
 
 def friend_signs_in
+  sleep 2
```

Comment:
[This](https://nulogy.com/articles/sleep-is-for-the-weak#.Vm8AUo0rLRY) is a good write up on the subject.